shelly.toml

```diff
@@ -7,4 +10,5 @@ cmdlets = [
     "Disconnect-VBRServer",
     "Start-VBRZip",
     "Find-VBRViEntity",
+    "vrfstats",
```

Comment:
Why ?

Reply:
1. PowerShell static analyzer which we use in CI cannot detect unknown cmdlets, because PowerShell is very dynamic language and it's impossible to statically know all the functions in scope.
2. Shelly was created to overcome that limitation, as we're using PowerShell in quite a disciplined way, so that kind of static analysis is possible.
3. Because of that, we need to know all the built-in functions that are not defined by us. Shelly contains a list of builtins, additionaly extended by this list in config.
4. It's impossible to differentiate syntactically between a command or a PowerShell function. Previous heuristic was you treat only `Verb-Foo` as function, but that's very limiting because analysis ignored single-word functions. Current heuristic is "everything not ending with .exe" is a function, therefore `vrfstats` got catched.
5. Anyway, I think current heuristic is fine and it's worth documenting usage of external command that way.

Do you think we should append the `.exe` to silence Shelly instead? Or perhaps introduce separate list, `commands` in Shelly.toml to avoid mixing with cmdlets?